### PR TITLE
#77 Upgrade material-ui

### PR DESCRIPTION
**Pull request by viettrung9012**
**Issue:** #77 
`theme.spacing.unit` will be deprecated in v5, so I followed the recommended update here https://github.com/mui-org/material-ui/tree/master/packages/material-ui-codemod/README.md#theme-spacing-api

### CODE REVIEW

**Comment by viettrung9012** [comment]
@codejockie tests passed in circleci but there is an error with missing token

File: yarn.lock [hunk 1]
```diff
@@ -1152,65 +1157,81 @@
...
+    "@types/react" "*"
 
-"@material-ui/utils@^3.0.0-alpha.2":
-  version "3.0.0-alpha.3"
-  resolved "https://registry.yarnpkg.com/@material-ui/utils/-/utils-3.0.0-alpha.3.tgz#836c62ea46f5ffc6f0b5ea05ab814704a86908b1"
-  integrity sha512-rwMdMZptX0DivkqBuC+Jdq7BYTXwqKai5G5ejPpuEDKpWzi1Oxp+LygGw329FrKpuKeiqpcymlqJTjmy+quWng==
+"@material-ui/utils@^4.1.0", "@material-ui/utils@^4.4.0":
+  version "4.4.0"
+  resolved "https://registry.npmjs.org/@material-ui/utils/-/utils-4.4.0.tgz#9275421e2798a067850d201212d46f12725828ad"
+  integrity sha512-UXoQVwArQEQWXxf2FPs0iJGT+MePQpKr0Qh0CPoLc1OdF0GSMTmQczcqCzwZkeHxHAOq/NkIKM1Pb/ih1Avicg==
```
Comment (codejockie) [inline yarn.lock]: @viettrung9012 it seem you used npm to install the updates. Please ensure you are using yarn for this project.

File: yarn.lock [hunk 2]
```diff
@@ -872,16 +872,16 @@
...
   resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.4.3.tgz#79888e452034223ad9609187a0ad1fe0d2ad4bdc"
   integrity sha512-9lsJwJLxDh/T3Q3SZszfWOTkk3pHbkmH+3KY+zwIDmsNlxsumuhS2TH3NIpktU4kNvfzy+k3eLT7aTJSPTo0OA==
   dependencies:
     regenerator-runtime "^0.13.2"
 
-"@babel/runtime@^7.4.0", "@babel/runtime@^7.5.5":
+"@babel/runtime@^7.3.1", "@babel/runtime@^7.4.0", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.2":
   version "7.6.2"
-  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.6.2.tgz#c3d6e41b304ef10dcf13777a33e7694ec4a9a6dd"
+  resolved "https://registry.npmjs.org/@babel/runtime/-/runtime-7.6.2.tgz#c3d6e41b304ef10dcf13777a33e7694ec4a9a6dd"
```
Comment (codejockie) [inline yarn.lock]: Please use yarn to install packages.

**Comment by viettrung9012** [comment]
updated the lock file
I did run yarn to add the package, but maybe due to my global registry setting

**Comment by codejockie** [comment]
@viettrung9012 there is a new version of material ui, I think 4.5.0. Please could you update this? Thank you.

**Pull request merged**